salt/modules/virt.py

```diff
@@ -3466,6 +3610,7 @@ def _parse_caps_guest(guest):
             'machines': {},
             'domains': {}
         },
+        'features': {}
```

Comment:
This will likely break the capabilities unit tests, and I'm not even sure it is needed with the next hunk